### ISSUE TITLE
Created a simple openzwave control panel with Node-Red-Dashboard

### ISSUE THREAD

**Issue by robertsLando** [OpenZWave/node-red-contrib-openzwave#44]
Here is the link to the flow: https://flows.nodered.org/flow/41ae75dbfbe6465d5cf40248e2500e0a

Enjoy :smile:

**Comment by ekarak** [comment]
Nice! I think of adding it into the documentation for people to quickly get started with OpenZWave on NodeRed. Can you please add some links to the extra nodes you're using (like the ui-stuff)? Even better, can you open up a pull request with instructions for beginners as how to install these extra nodes?

**Comment by robertsLando** [comment]
'ui-stuff' nodes are [node-red-dashboard](https://github.com/node-red/node-red-dashboard) nodes, the official dashboard of node-red. It can be installed from node-red manage-palette or by simply run `npm install node-red-dashboard` command in your node-red install directory.

**Comment by ekarak** [comment]
you're right. It seems that the node-red-contrib-ui is deprecated in favour of node-red-dashboard. 
I've now got your flow running (under Stretch btw) but I'm still having some (css?)  issues, as the node  template won't play the valueIDs:
![default](https://user-images.githubusercontent.com/587817/34074337-4b523f38-e2b5-11e7-98d6-95fe61a52445.png)

**Comment by robertsLando** [comment]
Are you using last node-red-dasboard and node-red version? I think the problem maybe is the dimension of the widget that contains the ui-template. Check that the dimensions of ui-template is not 'Auto' but a fixed value (try with 18x24) because dimension is calculated based on ui-template content but at init template has no content and so the dimension is 0x0 than it is not updated. To set the width of the widget > 6 you have to check that the ui-group that contains the ui-widget has the width you need (default it's 6)
